public/js/navigator-js/integration/ReactRecipe.js

```diff
@@ -0,0 +1,86 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var ReactRecipe = {
+  _type: 'REACT',
+
+  // Returns a fake proxy element due to the fact that
+  // we do not use these transition functions within
+  // our react components
+
+  getViewInstance: function getViewInstance() {
+    return {
+      navigatorBehaviors: ['IHasStateTransition'],
+      transitionIn: function(cb) { cb() },
```

Comment:
Yes we are.  I really wanted to make `transitionIn` and `transitionOut` work with React but there are some complications that would require a more involved refactor to Navigator due to the fact that react component references are not available immediately after `render` is called.

My proposal is that we start with this and address this issue in a follow up PR.